### DISCUSSION TITLE
Other repos not working with custom domain.

### DISCUSSION THREAD

**Discussion by AgedLace** [community/community#38896] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I have successfully set up my custom domain repo (https://benotdeceived.org) It is my understanding that my other repositories would be concatenated behind this custom domain. For example ... (https://benotdeceived.org/Bible-Study-Notebook)

None of my other repositories is available this way. I do not know what I am doing wrong.

**Comment by radar-blip** [comment]
All your content must be in the same repository. The other repositories are seperate projects.

**Reply by AgedLace** [reply]
I have seen tutorials saying this is possible.  Thank you for taking the time to respond!

**Comment by airtower-luna** [comment]
> It is my understanding that my other repositories would be concatenated behind this custom domain.

That only applies if the repository with the custom domain is what pages [calls your "user site"](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#types-of-github-pages-sites). A user site is one in a repository named in the pattern `<username>.github.io`.

Otherwise the custom domain is just for that repository.

**Reply by AgedLace** [reply]
I thought that it was, but I'll double check it.  Thank you!

**Reply by AgedLace** [reply]
I have confirmed, the custom domain is my user site:  https://agedlace.github.io/Be-Not-Deceived/

**Comment by AgedLace** [comment]
I think I've discovered where I'm going wrong.  I'll research it and get back.

Thank you all for responding!

**Reply by radar-blip** [reply]
Alright. Hope you're making progress.

**Comment by AgedLace** [comment]
Thought I solved my issue; unfortunately, it didn't work.  

When I read @airtower-luna's reminder that an account can only have 1 user / organization site, I changed my second repo into a subdomain, created the requisite CNAME record and it seemed to work exactly the way I want it to, but when I push changes to either repo, it breaks.  

Still researching.

**Comment by AgedLace** [comment, marked as answer]
FINALLY fixed this!  I have my user site on the custom domain, with the second repo automatically being suffixed to the custom domain name.
To be honest, I don't know what finally fixed it.  Probably typos.  I have *very* poor vision, and I'm constantly having issues due to it.

Again, thank you to all who have offered to help!  I appreciate it **very** much!
